src/components/AnnouncementBanner.tsx

```diff
@@ -34,15 +34,15 @@ const AnnouncementBanner = () => {
                   <GrAnnounce className="h-6 w-6" aria-hidden="true" />
                 </span>
                 <p className="ml-3 font-medium text-black">
-                  Thanks for your interest in the PSM launch!{' '}
+                  Thanks for your interest in PSM launch!{' '}
```

Comment:
stray deletion?

Reply:
I copied this word for word from the issue but I think it should go back in. Also added another link as just requested in slack. PTAL